presto-plugin-toolkit/src/main/java/io/prestosql/plugin/base/security/TableAccessControlRule.java

```diff
@@ -31,9 +36,11 @@
             Optional.empty(),
             Optional.empty(),
             Optional.empty(),
+            Optional.empty(),
             Optional.empty());
 
     private final Set<TablePrivilege> privileges;
+    private final Set<String> restrictedColumns;
```

Comment:
put columns under `tableRegex`

Reply:
The sorting is "permission" and then "matching"